### ISSUE TITLE
npm run develop, not work as expected

### ISSUE THREAD

**Issue by calcaide** [amwmedia/node-plop#3]
The problem occurs when you run `$ npm run develop` for second time. I clone the project, I run `npm run develop` and all is alright, but when I run it for a second time, the console is showing me a error:

```bash
loudRejection/api is deprecated. Use the currently-unhandled module instead.
module.js:327
    throw err;
    ^

Error: Cannot find module '../lib/index.js'
```

I will try to figure out what is happening.

**Comment by amwmedia** [comment]
Probably a stupid question, but I assume you've done 'npm install' and 'npm update'?

**Comment by calcaide** [comment]
Yes, that's right, I tried. Today I have no more time to spend in this, the next week I will take a look and try to solve the problem. Thanks

**Comment by amwmedia** [comment]
no problem! I appreciate your willingness to help :+1:

**Comment by amwmedia** [comment]
I just updated a bunch of dependencies and refactored the `generator-runner` code, try pulling down the latest code and let me know if you still have problems developing on it.

**Comment by calcaide** [comment]
Thanks, already I merge your changes to my fork, I run the `npm run develop` command and the issue described above persists. Now I will spend some time trying to figure out what is happening.

**Comment by amwmedia** [comment]
k, that's weird, I see a similar issue at first, but then the babel compile finishes and the tests re-run successfully.

**Comment by amwmedia** [comment]
this is what I see

![](https://www.evernote.com/l/AAH0pkPHybxLHbOVUnw3YmTUo4Vdu9aHsQQB/image.png)

**Comment by amwmedia** [comment]
I think I [fixed it](https://github.com/amwmedia/node-plop/commit/06447b844610dad39bd6c6ee0e4d2d4ecf34fea6).
